WordPress/build.gradle

```diff
@@ -301,7 +301,10 @@ dependencies {
     implementation "com.google.dagger:dagger-android-support:$daggerVersion"
     kapt "com.google.dagger:dagger-android-processor:$daggerVersion"
 
-    implementation ("com.github.wordpress-mobile.WordPress-FluxC-Android:fluxc:$fluxCVersion") {
+    def fluxCDependency = project.findProperty("localFluxC")?.toBoolean()
```

Comment:
I forgot to bring this up in the PR description yesterday, but I was a bit concerned about the `localFluxC` property name. We need a consistent naming for these included build properties and I don't know how much I like `localFluxC` as the naming pattern. Any ideas on that @mokagio @loremattei @jkmassel ?

Reply:
Good point. And possibly something to address sooner rather than later, because if the devs start adopting this it might become painful to migrate.

This might be overboard, but maybe the `included-builds` file could become more of a config file in YAML or something similar that defines the local dependencies setup for the user using keys which then `build.gradle` can look up.

```yml
local_dependencies: 
  fluxc:
    path: ../WordPress-FluxC-Android
  babanakit:
    path: ~/fridge/banana_kit
```

Then this code would check if the expected key for FluxC exists in the config YAML instead of looking for a bool value. The code in `settings.gradle` would be a bit more complex, but basically for each key check if the path exists and if so `includeBuild` on it.

Reply:
@mokagio Thank you for this! I didn't end up going this exact solution, but your input made me re-consider my assumptions and I ended up going with something different which hopefully still caries out your intent. I'd love to get your input again after the latest changes.

Reply:
Thanks @oguzkocer, glad my input was useful 😄 

I had a look here and it looks good. It's very similar to my YAML suggestion, but more Gradle-y, which is probably a good thing.